portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function goToHomePage() {
+    location.href = "index.html";
```

Comment:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Ordinary string literals are delimited with single quotes ('), rather than double quotes (").

Reply:
Same throughout the js files